src/material/core/ripple/ripple-renderer.ts

```diff
@@ -65,8 +71,13 @@ export class RippleRenderer implements EventListenerObject {
   /** Whether the pointer is currently down or not. */
   private _isPointerDown = false;
 
-  /** Set of currently active ripple references. */
-  private _activeRipples = new Set<RippleRef>();
+  /**
+   * Map of currently active ripple references.
+   * The ripple reference is mapped to its element event listeners.
+   * The reason why `| null` is used is that event listeners are added only
+   * when the condition is truthy (see the `_startFadeOutTransition` method).
```

Comment:
Not sure if `_startFadeOutTransition` is the best reference here, but thanks for the comment 👍